src/unix/solarish/mod.rs

```diff
@@ -420,6 +420,24 @@ s! {
         pub portnfy_user: *mut ::c_void,
     }
 
+    pub struct aio_result_t {
+        aio_return: ::ssize_t,
```

Comment:
Why are these fields not `pub`?  They're mentioned in [aioread(3C)](https://illumos.org/man/3C/aioread) so I feel like people ought to be able to access them.

Reply:
Oh, my fault. Fixed.